### PR TITLE
update 090 to new common files

### PR DESCRIPTION
**Pull request by iCollin**
This PR is **ready** for review.

### Summary
this test was a non-actual failure. i've updated it to use the new common files so this test will be valid.

### ATF version
latest

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/Policies/build_options/090_ATF_For_PTU_use_consented_device_only.lua [hunk 1]
```diff
@@ -18,165 +18,86 @@
...
-commonFunctions:SDLForceStop()
-commonSteps:DeleteLogsFileAndPolicyTable()
-
---[[ Local variables ]]
-local mobileHost = "1.0.0.1"
-local deviceMAC2 = "9cc72994ab9ca68c1daaf02834f7a94552e82aad3250778f2e12d14afee0a5c6"
-local deviceName2 = mobileHost .. ":" .. config.mobilePort
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/TheSameApp/commonTheSameApp')
+local common2 = require('test_scripts/Capabilities/PersistingHMICapabilities/common')
```
Comment (jacobkeeler) [inline test_scripts/Policies/build_options/090_ATF_For_PTU_use_consented_device_only.lua:24]:
```suggestion

--[[ Test Configuration ]]
runner.testSettings.isSelfIncluded = false
```
common2 shouldn't be needed, I think the issue is that `isSelfIncluded = false` is missing.

File: test_scripts/Policies/build_options/090_ATF_For_PTU_use_consented_device_only.lua [hunk 2]
```diff
@@ -18,165 +18,86 @@
...
+          common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
         end)
     end)
 end
 
---[[ Postconditions ]]
-commonFunctions:newTestCasesGroup("Postconditions")
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI", common2.start)
```
Comment (jacobkeeler) [inline test_scripts/Policies/build_options/090_ATF_For_PTU_use_consented_device_only.lua:91]:
```suggestion
runner.Step("Start SDL, HMI", common.start)
```

File: test_scripts/Policies/build_options/090_ATF_For_PTU_use_consented_device_only.lua [hunk 3]
```diff
@@ -18,165 +18,86 @@
...
-    end)
+    common.connectMobDevice(2, devices[2], false)
 end
 
-function Test:Teat_Activate_app_2()
-  local RequestId = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = self.HMIAppID2})
-  EXPECT_HMIRESPONSE(RequestId, {result = { code = 0, device = { id = deviceMAC2, name = deviceName2 }, isSDLAllowed = false, method ="SDL.ActivateApp" }})
+local function activateAppTwo()
+  local RequestId = common.getHMIConnection():SendRequest("SDL.ActivateApp", { appID = common.getHMIAppId(2) })
+  EXPECT_HMIRESPONSE(RequestId, {result = { code = 0, device = { name = devices[2].name }, isSDLAllowed = false, method = "SDL.ActivateApp" }})
```
Comment (jacobkeeler) [inline test_scripts/Policies/build_options/090_ATF_For_PTU_use_consented_device_only.lua:64]:
```suggestion
  common.getHMIConnection():ExpectResponse(RequestId, {result = { code = 0, device = { name = devices[2].name }, isSDLAllowed = false, method = "SDL.ActivateApp" }})
```
for consistency

File: test_scripts/Policies/build_options/090_ATF_For_PTU_use_consented_device_only.lua [hunk 4]
```diff
@@ -18,165 +18,86 @@
...
-          self.hmiConnection:SendNotification("SDL.OnAllowSDLFunctionality",
-            {allowed = false, source = "GUI", device = {id = deviceMAC2, name = deviceName2}})
-          EXPECT_HMICALL("BasicCommunication.ActivateApp")
-          :Do(function(_, data)
-              self.hmiConnection:SendResponse(data.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
-            end)
-          :Times(AtLeast(1))
-        end)
+    local RequestIdGetMes = common.getHMIConnection():SendRequest("SDL.GetUserFriendlyMessage", { language = "EN-US", messageCodes = { "DataConsent" } })
+    EXPECT_HMIRESPONSE(RequestIdGetMes)
```
Comment (jacobkeeler) [inline test_scripts/Policies/build_options/090_ATF_For_PTU_use_consented_device_only.lua:67]:
```suggestion
    common.getHMIConnection():ExpectResponse(RequestIdGetMes)
```

File: test_scripts/Policies/build_options/090_ATF_For_PTU_use_consented_device_only.lua [hunk 5]
```diff
@@ -18,165 +18,86 @@
...
 end
 
-function Test:Test_Start_PTU()
-  EXPECT_HMICALL("BasicCommunication.PolicyUpdate")
-  :Do(function()
-      self.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest",
-        { requestType = "PROPRIETARY", fileName = "PolicyTableUpdate", appID = self.HMIAppID })
-      self.mobileSession2:ExpectNotification("OnSystemRequest", {requestType = "PROPRIETARY"})
+local function startPTU()
+    EXPECT_HMICALL("BasicCommunication.PolicyUpdate")
```
Comment (jacobkeeler) [inline test_scripts/Policies/build_options/090_ATF_For_PTU_use_consented_device_only.lua:76]:
```suggestion
    common.getHMIConnection():ExpectRequest("BasicCommunication.PolicyUpdate")
```

**Review by jacobkeeler — CHANGES_REQUESTED** [review]

**Comment by iCollin** [comment]
Great catch on `isSelfIncluded`, all comments were adressed in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2479/commits/669c8680a255840c488d74d164f59b9988e56ccb

**Comment by dboltovskyi** [comment]
@iCollin Please consider 2 items from my side:
1. In order to support (and restrict) more connection types and SDL configurations I would suggest to make updates similar the ones described in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2485
2. Add script into policy test set: `policies_happy_paths_EXTERNAL_PROPRIETARY.txt` and enable it in `policies_all_flows.txt`

**Review by jacobkeeler — APPROVED** [review]

**Pull request merged**